pkg/aws_keyhub/config.go

```diff
@@ -42,7 +41,7 @@ func ConfigureAwsKeyhub() {
 		KeyHubUrl             string
 		KeyHubClientId        string
 		KeyHubAwsSamlClientId string
-		AssumeDuration        int64
+		AssumeDuration        int32
```

Comment:
Wat is de reden dat deze kleiner is?

Reply:
AWS heeft  deze variabele in zijn sdk kleiner gemaakt.